pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -2355,6 +2361,12 @@ DOHFrontend
 
     :param str ticketsKeysFile: The path to a file from where TLS tickets keys should be loaded.
 
+  .. method:: DOHFrontend:loadTicketsKey(key)
+
+     Replace the current TLS tickets key with a given one.
```

Comment:
Technically it adds a new key, making the existing one inactive but still usable to decrypt existing tickets (at least for the OpenSSL implementation, I can't remember if we do this for the GnuTLS one).
```suggestion
     Load a new TLS tickets key.
```

Reply:
There is only one key in the context of GnuTLS, but you are right for OpenSSL.